client/e2e/testData/dataConstants/EventInfoConstants.ts

```diff
@@ -14,6 +15,7 @@ export const EVENT_INFO_2: EventInfo = {
     'https://www.hindustantimes.com/ht-img/img/2023/08/25/550x309/international_dog_day_1692974397743_1692974414085.jpg',
   eventFullName:
```

Comment:
Similarly, dont have a full name that has the string "Playbill" in here.

Reply:
create the string, using event name and description, within the page function that needs this weird string. this will ensure consistency and minimize errors.